liquidator/liquidator.js

```diff
@@ -26,8 +40,22 @@ class Liquidator {
 
   // Queries underCollateralized positions and performs liquidations against any under collateralized positions.
   queryAndLiquidate = async priceFunction => {
+    const { toBN, fromWei } = this.web3.utils;
+
     const contractTime = this.empClient.getLastUpdateTime();
-    const priceFeed = priceFunction(contractTime);
+    let priceFeed = priceFunction(contractTime);
+
+    // The `priceFeed` is a Number that is used to determine if a position is liquidatable. The higher the
+    // `priceFeed` value, the more collateral that the position is required to have to be correctly collateralized.
+    // Therefore, we add a buffer by reducing `priceFeed` to (`crThreshold` * `priceFeed`).
+    priceFeed = fromWei(toBN(priceFeed).mul(toBN(this.crThreshold)));
+
+    this.logger.debug({
+      at: "Liquidator",
+      message: "Raising liquidation price threshold",
+      priceFeed: priceFeed,
```

Comment:
Here, for example, you can see where this becomes hard to reason about re my comment above. `priceFeed` is no longer the price of the data identifier; it;s the scaled value. but it's called priceFeed. To get the price from only looking at the slack message for example you need to divide the price feed by the `priceThreshold`. Re-naming this would help.

Reply:
makes sense, I'll go with the `scaledPriceFeed` suggestion